src/models/militaryDiscipline.js

```diff
@@ -0,0 +1,9 @@
+const militaryDiscipline = {
+  Date: { presence: true, date: true },
```

Comment:
Based on the UI I think this _should_ use `date: { requireDay: false }`. The validator is working without it because date control seems to pass a default value for day if the day field is hidden, but I think it'd be good to explicitly include this flag so it's not dependent on that.